x-pack/plugins/observability_solution/infra/public/components/asset_details/hooks/use_loading_state.ts

```diff
@@ -37,16 +36,11 @@ export const useLoadingState = () => {
   const {
     data: { search },
   } = services;
-  const { updateSearchSessionId } = useSearchSessionContext();
```

Comment:
We'll refactor this context to be timestamp-based instead of relying on searchSessionId, but removing this now will have consequences beyond the embedabbles. This context is useful to control and force requests